courses/machine_learning/deepdive2/supplemental/solutions/object_detection_with_automl.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<ul><li>"and where the object was found in the image" -&gt; "and its location within the image"</li><li>Maybe add a description of the whole workflow (image upload, training, evaluation, deployment, etc.) as well as the dataset to be used: e.g., "We will start by uploading to AutoML Vision a image dataset consisting of ..."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>done</p>